packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -348,6 +361,9 @@ export class RunningSummarizer implements IDisposable {
 		this.runtime.deltaManager.off("op", (op) => {
 			this.handleOp(op);
 		});
+		this.runtime.off?.("op", (op, runtimeMessage) => {
```

Comment:
This looks wrong (same for above) -  this creates new listner object, and thus calling off() with it will do nothing as this listener instance is not registered.